### DISCUSSION TITLE
API values

### DISCUSSION THREAD

**Discussion by Depechie** [pypolestar/polestar_api#30] [General]
I'm still on 1.0.2 and not getting anything for charging current and charging power.

This is normal behaviour?

**Comment by leeyuentuen** [comment]
this is issue from polestar for not providing the information. but that key was included in the response code

**Reply by Depechie** [reply]
Ok got it! Thx for the clarification.

**Comment by Depechie** [comment]
With version 1.2.1 the model name and vin also became 'unknown' But had values before?

**Reply by leeyuentuen** [reply]
need to check, but sees a reload of the integration solved temp. the issue

**Reply by leeyuentuen** [reply]
i see the issue, it should be fixed in 1.2.4, normally there was a skip cache, but didn't work well in the previous version
https://github.com/leeyuentuen/polestar_api/releases/tag/1.2.4

**Comment by leeyuentuen** [comment]
can you take a look at the beta version 1.2.3?
